### PR TITLE
Add netlify build command that includes yarn setup

### PR DESCRIPTION
**Pull request by CodeByAlex**
## What I did
Added a command for netlify to use to setup the project
## How to test

- Is this testable with Jest or Chromatic screenshots?
- Does this need a new example in the kitchen sink apps?
- Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,11 +10,15 @@
   "scripts": {
     "build": "yarn build:tsc",
     "build:tsc": "lerna exec --scope @storybook/* --parallel -- cross-env-shell node \\$LERNA_ROOT_PATH/scripts/watch-tsc.js",
+    "build-storybook": "build-storybook",
+    "change-directory-to-example": "cd ./examples/aem-kitchen-sink",
+    "netlify-build": "npm-run-all yarn change-directory-to-example change-directory-to-example build-storybook",
```
Comment (tripodsan) [inline package.json:15]:
```suggestion
    "netlify-build": "npm-run-all yarn change-directory-to-example build-storybook",
```

File: package.json [hunk 2]
```diff
@@ -10,11 +10,15 @@
   "scripts": {
     "build": "yarn build:tsc",
     "build:tsc": "lerna exec --scope @storybook/* --parallel -- cross-env-shell node \\$LERNA_ROOT_PATH/scripts/watch-tsc.js",
+    "build-storybook": "build-storybook",
```
Comment (tripodsan) [inline package.json:13]:
```suggestion
    "build-storybook": "yarn storybook",
```

Comment (CodeByAlex) [reply package.json:13]: It should be build storybook to create the static instance

**Comment by CodeByAlex** [comment]
Was not working as expected - will reopen when ready

File: package.json (hunk 2 above)
Comment (tripodsan) [reply package.json]: ok. it looked like a typo :-)

Comment (CodeByAlex) [reply package.json]: No worries - thanks for looking

**Pull request merged**